article/related-work.tex

```diff
@@ -10,13 +10,48 @@ \subsection{Light GCN}
 Light GCN also shows that NGCF will significantly improve if both the activation function and feature transformation are removed \cite{lightgcn}.
+
+\subsection{Price-aware Recommendation with Graph Convolutional Networks}
+Yu Zheng et al. develops an effective method called \textit{Price-aware User Preference-modeling (PUP)} that is used to create recommendations with focus on the price factor \cite{Priceaware}.
+There are two main challenges where the first is that the preferences of a user on item price are unknown, and can only be seen implicitly by previous purchases.
+The second challenge is that the price of an item largely depends on what category the item is within.
+They solve this by integrating the categories and items into the propagation process.
+PUP represents the data as an heterogeneous undirected graph $G = (V,E)$ where the nodes in $V$ consists of user nodes $u \in U$, item nodes $i \in I$, category nodes $c \in \textbf{c}$, and price nodes $p \in \textbf{p}$.
+The edges in $E$ consists of interaction edges $(u, i)$ with $R_{ui} = 1$ if there is an interaction between user $u$ and item $i$ and category edges $(i, \textbf{c}_i)$ and price edges $(i, \textbf{p}_i)$.
+Traditional Latent Factor Models such as Matrix Factorization is with this graph insufficient for Recommendation due to being more pairs than $(u,i)$, but also $(i,p)$ and $(i,c)$ \cite{Priceaware}.
+Hereby Yu Zheng et al. use Graph Neural Network to learn the embeddings so that each node has a separate embedding $e' \in \mathbb{R}^d$ where d is the dimensions of the embedding.
+In GCN the nodes propagate its nearest neighbors, which could be user-item, item-price or item-category.
+Embeddings from node j to node i are propagated as follows:
+\begin{equation}
+    t_{ji} = \frac{1}{|\mathcal{N}_i|}e'_j,
+\end{equation}
+where $\mathcal{N}_i$ is the set of neighbors for node $i$ and $e'_j$ is the embedding node j.
+The updating rule is:
+\begin{align*}
+     & o_u = \sum_{j \in \{i \textrm{ with } R_{ui}=1 \} \cup \{ u\}}^{} t_{ju}                            \\
+     & o_i = \sum_{j \in \{u \textrm{ with } R_{ui}=1 \} \cup \{ i, \textbf{c}_i \textbf{p}_i\}}^{} t_{ji} \\
+     & o_c = \sum_{j \in \{i \textrm{ with } \textbf{c}_i=c \} \cup \{ c\}}^{} t_{jc}                      \\
+     & o_p = \sum_{j \in \{i \textrm{ with } \textbf{p}_i=c \} \cup \{ p\}}^{} t_{jp}                      \\
+     & e_f = \textrm{tanh}(o_f), f \in \{u, i, c, p\},
+\end{align*}
+where $e_u$, $e_i$, $e_c$, and $e_p$ are the embeddings for user $u$, item $i$, category $c$ and price $p$.
+The intuition about this updating rule is that the nodes gets aggregated together with its neighbors and in each layer it will go one step further.
```

Comment:
```suggestion
The intuition about this updating rule is that the node gets aggregated together with its neighbors and in each layer it will go one step further.
```